src/todos.js

```diff
@@ -0,0 +1,37 @@
+let express = require('express');
+let bodyParser = require('body-parser');
+
+let app = express();
+app.use(bodyParser.json());
+
+let todos = [{id: 'jkhsdjkf', content: 'review this code'}];
+
+app.post('/todos', (req, res) => {
+  todos.push({
+    ...req.body,
+    id: Math.random().toString(32).slice(2)
```

Comment:
1. Do not use `Math.random()` to generate unique `id`s. Do not use an incremented counter either, as it could lead to potential security issues in the future, if we ever want to make todos ids secret. Use [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier)s.
2. Use `id` as key as per my comment at L7.